src/main/controllers/SubscriptionConfigListLanguageController.ts

```diff
@@ -0,0 +1,27 @@
+import { PipRequest } from '../models/request/PipRequest';
```

Comment:
We should rename this file and all related view and language files to SubscriptionConfigureLIstLanguage and not SubscriptionConfigLIstLanguage to be consistent with other files

Reply:
fixed.